tutorcelery/patches/k8s-override

```diff
@@ -0,0 +1,22 @@
+{% for service in ["lms", "cms"] %}
+
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{service}}-worker
+spec:
+  template:
+    spec:
+      containers:
+        - name: {{service}}-worker
+          image: {{ DOCKER_IMAGE_OPENEDX }}
+          args:
+            - "celery"
+            - "--app={{service}}.celery"
+            - "worker"
+            - "--loglevel=info"
+            - "--hostname=edx.{{service}}.core.default"
```

Comment:
```suggestion
            - "--hostname=edx.{{service}}.core.default.%%h"
```
Should it be like the [Tutor hostname](https://github.com/overhangio/tutor/blob/master/tutor/templates/k8s/deployments.yml#L144C110-L144C113)?